src/components/AddressSummary.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import ReactLoading from 'react-loading';
+import { Link } from 'react-router-dom';
+import {TX_COUNT} from '../constants';
+import TxRow from './TxRow';
+import helpers from '../utils/helpers';
+import WebSocketHandler from '../WebSocketHandler';
+import colors from '../index.scss';
+import hathorLib from '@hathor/wallet-lib';
+import PropTypes from 'prop-types';
+
+
+class AddressSummary extends React.Component {
+  /**
+   * balance {Object} Object with balance of each token of this address indexed by tokenUid {uid1: {'received', 'spent'}}
+   * quantity {number} Number of transactions in this address
+   * loading {boolean} If is waiting server response
+   * errorMessage {String} Message to be shown in case of an error
+   * selectedToken {String} UID of the selected token when address has many
+   */
+  state = {
+    balance: {},
+    quantity: null,
```

Comment:
I'd rename it to `number_of_transactions`, or `n_transactions`, or `transactions_count`. What do you think?

Reply:
This variable does not exist anymore